packages/hebao_v1/contracts/modules/security/GuardianUtils.sol

```diff
@@ -44,17 +46,24 @@ library GuardianUtils
             require(signers[i] > lastSigner, "INVALID_SIGNERS_ORDER");
             lastSigner = signers[i];
 
-            if (signers[i] == owner) {
+            if (signers[i] == owner || signers[i] == wallet) {
                 walletOwnerSigned = true;
```

Comment:
I think this needs a `require(!walletOwnerSigned, "WALLET_CANNOT_SIGN_TWICE")` here. Now the owner of the wallet can take up 2 slots in `signers.length`, and so the majority calculation will be wrong. So easiest I think to just limit it here so only one of the 2 is used.

Reply:
This is really a nice catch!